tests/assets/test-nb-inject-assets/template_b.ipynb

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
Is there an existing pipeline we can leverage here?

Reply:
I think there are some pipelines we can use but it might be confusing to debug/create more tests in the future since we need to check a lot of different cases with this injection (this is why I created a specific assets folder for nb-injection)